app/webpacker/stylesheets/components/pages/certification/_cs-accelerator.scss

```diff
@@ -18,11 +18,23 @@
 
   @include govuk-media-query($from: desktop) {
     flex-direction: row;
+    .icon-dotted-line {
+      flex-shrink: 0;
+    }
   }
 }
 
 .ncce-progress-section--text {
-  width: 200px;
+  flex-grow: 1;
+  min-height: 3rem;
+
+  display: flex;
+  align-items: center;
+
+  @include govuk-media-query($from: desktop) {
+    background-position-y: 8px;
+    display: block;
+  }
 }
```

Comment:
It is a tiny nit that is just a legacy from the code, that this page's filename suggests that it relates to the CSA, whereas I expect its classes are used on all 4 (soon to be 5) certificate dashbaords.

Reply:
For some reason the styles for the progress bar were added in this CSS file. I've moved them into their own.